account_billing_adjust_cft/models/account_billing.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ecosoft Co. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api, _
+
+
+class AccountBillingAdjust(models.Model):
+    _inherit = 'account.billing'
+
+    @api.multi
+    def onchange_partner_id(self, company_id,
+                            partner_id, currency_id, date):
+        ctx = self._context.copy()
+
+        if not currency_id:
+            return {'value': {'line_cr_ids': []}}
+        res = self.with_context(ctx).recompute_billing_lines(
+            company_id, partner_id, currency_id, date)
+        return res
```

Comment:
Make the method like below including comments:

```.py
    # overriding the method to remove billing_date_condition
    # so that all the outstanding ARs for the customer are proposed
    @api.multi
    def onchange_partner_id(self, company_id,
                            partner_id, currency_id, date):
        # ctx = self._context.copy()
        # ctx.update(
        #     {'billing_date_condition': ['|',
        #                                 ('date_maturity', '=', False),
        #                                 ('date_maturity', '<=', date)]}
        # )
        if not currency_id:
            return {'value': {'line_cr_ids': []}}
        # res = self.with_context(ctx).recompute_billing_lines(
        res = self.recompute_billing_lines(
            company_id, partner_id, currency_id, date)
        return res
```